### DISCUSSION TITLE
This experimental syntax requires enabling the parser plugin: 'classProperties'

### DISCUSSION THREAD

**Discussion by DEVTomatoCake** [parcel-bundler/parcel#6028] [Q&A] [answered]
I got the same error as #5943 and tried all fixes, but force-downgrading the version of parcel-bundler didnt help, same for using the updated babel and updating to parcel v2.

I got the following error after updating to parcel v2:
![image](https://user-images.githubusercontent.com/60300461/111824616-bbd3af00-88e6-11eb-92e6-72f770678f06.png)

**Comment by mischnic** [comment, marked as answer]
- Do you have a babelrc?
- Are you sure you're using the latest version of preset-env?

**Comment by DEVTomatoCake** [comment]
Oh I was missing the .babelrc-file, so it starts now!
I've got another problem now but I look for a solution by myself first...
